ui/page/components/items_scroll.go

```diff
@@ -58,7 +60,8 @@ func NewScroll[T any](load *load.Load, pageSize int32, queryFunc ScrollFunc[T])
 
 // FetchScrollData is a mutex protected fetchScrollData function. At the end of
 // the function call a window reload is triggered. Returns that latest records.
-func (s *Scroll[T]) FetchScrollData(isReverse bool, window app.WindowNavigator) {
+// If reset is true, the offset value will be reset to 0.
+func (s *Scroll[T]) FetchScrollData(reset bool, window app.WindowNavigator) {
```

Comment:
Improve this comment. This is an exported method but it doesn't say what it does. Instead, it references a private method. Please update this comment with a clear description of what this method does.